README.md

```diff
@@ -40,19 +43,19 @@ microcloud init
 
 Following the simple CLI prompts, a working MicroCloud will be ready within minutes.
 
-The MicroCloud snap drives two other snaps (LXD and MicroCeph), enabling automated deployment of a highly available LXD cluster for compute with Ceph as a storage backend.
+The MicroCloud snap drives three other snaps (LXD, MicroCeph, and MicroOVN), enabling automated deployment of a highly available LXD cluster for compute with Ceph as the storage driver and OVN as the managed network.
 
-After the first initialisation steps, MicroCloud will detect the other servers, set up a cluster and finally prompt you to add disks to Ceph.
+During initialisation, MicroCloud detects the other servers and then prompts you to add disks to Ceph and configure the networking setup.
 
-At the end of this, you’ll have both a Ceph and a LXD cluster, and LXD itself will have been configured with both networking and storage suitable for use in a cluster.
+At the end of this, you’ll have an OVN cluster, a Ceph cluster, and a LXD cluster. LXD itself will have been configured with both networking and storage suitable for use in a cluster.
 
 ## **What about networking?**
 
-For networking, MicroCloud uses a default network bridge. **MicroOVN** is in development and will be added once completed.
+For networking, MicroCloud uses MicroOVN, a minimal implementation of OVN (Open Virtual Network).
 
 ## **What's next?**
 
-This is just the beginning of MicroCloud. We’re very excited about what’s coming up next, starting with the addition of OVN to the mix, providing distributed networking alongside the distributed storage provided by Ceph.
+This is just the beginning of MicroCloud. We’re very excited about what’s coming up next!
```

Comment:
Do we have any upcoming features we could mention here?

Reply:
Not at this time. We may add Kubernetes there later, but we've not locked that into the roadmap yet.